### ISSUE TITLE
Guake change position on toggle

### ISSUE THREAD

**Issue by AurelReb** [Guake/guake#2180]
**Describe the bug**

Guake terminal's position changes on every toggle, moving step by step to the bottom-right corner of my screen.
I get this bug since the last release of guake (3.10) installed with pacman.

Here is the output of `guake --support` :

<details><summary>$ guake --support</summary>

Guake Version:		3.11.dev0

Vte Version:		0.72.1

Vte Runtime Version:	0.72.1

--------------------------------------------------
GTK+ Version:		3.24.37

GDK Backend:		<GdkX11.X11Display

--------------------------------------------------
Desktop Session: i3

--------------------------------------------------
Display: :0

RGBA visual: True

Composited: True

* Monitor: 0 - AUO eDP-1
    * Geometry:		2560 x 1600 at 0, 1350
    * Size:		344 x 215 mm²
    * Primary:		True
    * Refresh rate:	60.03 Hz
    * Subpixel layout:	unknown
* Monitor: 1 - PHL HDMI-1
    * Geometry:		2400 x 1350 at 80, 0
    * Size:		477 x 268 mm²
    * Primary:		False
    * Refresh rate:	60.0 Hz
    * Subpixel layout:	unknown
</details>

I'm using archlinux

**Expected behavior**

Guake opens everytime at the same position on the screen

**Actual behavior**

First toggle:
![image](https://github.com/Guake/guake/assets/46166884/df0b12be-a434-4f1f-8d23-e66bcc3e941b)

After 7 toggles : 
![image](https://github.com/Guake/guake/assets/46166884/c1071407-f41b-4dfe-9648-63715da14921)

**To Reproduce**

Run guake version 3.10 with a fully updated archlinux system, with i3wm.

**Comment by trinaldi** [comment]
Same here. I have the same specs (i3wn + X11 + Arch).

Every time I hit F12 (toggle), the terminal drops a few pixels. Toggling fullscreen reverts the position.

I reverted back to 3.9 and added guake to ignore list on `pacman.conf`

**Comment by szmktk** [comment]
Same here - I encountered the same behavior on both of my machines:
- Arch, i3-wm 4.22, guake 3.10
- Ubuntu 20.04, i3-wm 4.22, guake 3.10

Thanks to `trinaldi` I mitigated this issue by downgrading, but fixing this issue would be most appreciated.

BTW this issue seems to be a duplicate of https://github.com/Guake/guake/issues/2177

**Comment by palytoxin** [comment]
I have two computers. (KDE/i3wm)

The new version has fixed the issue of width on KDE.
But i3wm also same problem as upon

https://github.com/Guake/guake/assets/1829328/1afe27b9-2de8-4265-adbf-76682418dfd8

**Comment by liqiang0330** [comment]
@trinaldi trinaldi  thank u. 

guake 3.10 have a bug.  downgrade to guake 3.9

1, sudo vim /etc/pacman.conf
add guake to ingore pkg.
2, sudo downgrade guake
choose guake 3.9

**Comment by trinaldi** [comment]
Piggybacking on @liqiang0330

You can get the [3.9 version from the Arch Archive](https://archive.archlinux.org/packages/g/guake/)

1. `sudoedit /etc/pacman.conf` and add `guake` to `IgnorePkg` (line 25 for me):

 ```
...
IgnorePkg   = guake
...
``` 

2. Inside the directory where you downloaded the `guake-3.9.0-3-any.pkg.tar.zst` file, run 

`sudo pacman -U guake-3.9.0-3-any.pkg.tar.zst`

**Comment by madhur** [comment]
I have switched to kitty terminal due to this bug

**Comment by Enr1g** [comment]
i3-wm fix without downgrading (`$HOME/.config/i3/config`):

```
for_window [class="Guake"] move window to position 0 px 0 px
```

**Comment by cowclaw** [comment]
Same issue here on: xubuntu 23.10, X11 21.1.7-3ubuntu2.1, i3 wm 4.22-2, guake 3.10-1

**Comment by elementary-charge** [comment]
i3-wm fix without downgrading ($HOME/.config/i3/config):

for_window [class="Guake"] move window to position 0 px 0 px

**Comment by elementary-charge** [comment]
Based on previous answer but for BOTTOM side

i3-wm fix without downgrading ($HOME/.config/i3/config):

for_window [class="Guake"] move window to position 0 px 50 ppt, set resize 100 ppt 50 ppt

**Comment by elementary-charge** [comment]
Based on previous answer for i3wm but for the BOTTOM screen side:

for_window [class="Guake"] move window to position 0 ppt 50 ppt, set resize 100 ppt 50 ppt

**Comment by novaTopFlex** [comment]
I believe that this may be an issue with your window manager or even your Xorg or Wayland configuration. I am currently using Xorg with Openbox as my window manager without a DE and do not have any issues with placement at all. On my Guake settings, my system respects my bottom-align desire, and the only issue that I have is with the `tint2` application running, and even then, changing tabs often reverts the position to no longer cover excessive width.
***
In simpler terms, I am not suffering from any slight movement of this Guake terminal between showing and hiding of the terminal with Openbox on Xorg.
